src/sync/discovery.ts

```diff
@@ -3,22 +3,54 @@ import { getRepositorySummary } from "./jobs";
 import enhanceOctokit from "../config/enhance-octokit";
 import { Application } from "probot";
 import { Repositories, SyncStatus } from "../models/subscription";
-import {LoggerWithTarget} from "probot/lib/wrap-logger";
+import { LoggerWithTarget } from "probot/lib/wrap-logger";
 import { sqsQueues } from "../sqs/queues";
+import GitHubClient from "../github/client/github-client";
+import { getCloudInstallationId } from "../github/client/installation-id";
+import { Repository } from "@octokit/graphql-schema";
+import { booleanFlag, BooleanFlags } from "../config/feature-flags";
 
 export const DISCOVERY_LOGGER_NAME = "sync.discovery";
+const MAX_PAGE_SIZE_REPOSITORY = 100;
```

Comment:
Set to 100 because same as  what Octokit had, but also there is an upper limit with the graphql query of 100